pkg/kube/pod.go

```diff
@@ -48,6 +48,11 @@ const (
 	defaultContainerName   = "container"
 )
 
+type VolumeMountOptions struct {
+	MountPoint string
```

Comment:
```suggestion
	MountPath string
```

Reply:
do we handle block devices as well, using the same struct, i.e., `VolumeMountsOptions`?

Reply:
I see, in K8s for volume mounts uses `MountPath` term, regardless in all POSIX-compatible OSes for naming directory where volume mounted to uses `MountPoint` term. Ok, I'll rename it.

Reply:
fixed